compiler/rustc_error_codes/src/error_codes/E0523.md

```diff
@@ -0,0 +1,4 @@
+#### Note: this error code is no longer emitted by the compiler.
+
+Found two different crates that are not distinguished by differing metadata.
+Code that previously produced this error will now produce E0464 instead.
```

Comment:
It'd be great if we could have a code example to show some code which previously produced this error here. You can just copy it from the E0523.md file.

Also a small nitpick: when you add a code example, the note explaining that this error has been merged into E0464 could be below the code example (I think that's more in line with how it's normally formatted).

Reply:
> It'd be great if we could have a code example to show some code which previously produced this error here. You can just copy it from the E0523.md file.

Sure.  I'm going to assume that you meant the `E0464.rs` file (based on an earlier comment), as `E0523.md` is this file and `E0464.md` doesn't include a code example.

> when you add a code example, the note explaining that this error has been merged into E0464

Done!  Let me know if I got the formatting right.